playbooks/templates/standalone_parameters.yaml.j2

```diff
@@ -57,9 +57,13 @@ parameter_defaults:
   NeutronBridgeMappings: {{ neutron_bridge_mappings }}
   NeutronFlatNetworks: {{ neutron_flat_networks }}
 {% if sriov_interface is defined %}
-  NeutronPhysicalDevMappings: "hostonly:{{ sriov_interface }}"
+  NeutronPhysicalDevMappings: "hostonly-sriov:{{ sriov_interface }}"
```

Comment:
Don't we also need to map `hostonly-dpdk` to `dpdk_interface` when `dpdk_interface` is defined?

Reply:
@christophefontaine can you please answer?